tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "verbatimModuleSyntax": true,
     "allowUnreachableCode": false,
     "allowUnusedLabels": false,
-    "exactOptionalPropertyTypes": true,
+    "exactOptionalPropertyTypes": false,
```

Comment:
This was a super weird one. It would fail optional component props. Things like `onClick?: () => void`

Reply:
agree